src/Microsoft.CodeAnalysis.Analyzers/Core/CodeAnalysisDiagnosticsResources.resx

```diff
@@ -269,7 +269,7 @@
     <value>'{0}' is immutable and '{1}' will not have any effect on it. Consider using the return value from '{1}'.</value>
   </data>
   <data name="DoNotIgnoreReturnValueOnImmutableObjectMethodInvocationTitle" xml:space="preserve">
-    <value>Do not ignore values returned by methods on immutable objects.</value>
+    <value>Do not ignore values returned by methods on immutable objects</value>
```

Comment:
This was reported by RS1031. I don't understand why it wasn't reported earlier, but there is something going wrong.

Reply:
I found the issue and will open an issue.